### PR TITLE
commands/publish_images: Added dockerbuild_context parameter.

### PR DESCRIPTION
**Pull request by geekodour**
Added `dockerbuild_context` parameter to set the `DOCKERBUILD_CONTEXT` environment variable.

Signed-off-by: Hrishikesh Barman <hrishikeshbman@gmail.com>

### CODE REVIEW

**Review by krasi-georgiev — APPROVED** [review]

File: src/commands/publish_images.yml [hunk 1]
```diff
@@ -18,6 +18,10 @@ parameters:
     description: Path to Dockerfile
     type: string
     default: ""
+  dockerbuild_context:
+    description: Path to Dockerbuild context
+    type: string
+    default: "./"
```
Comment (simonpasquier) [inline src/commands/publish_images.yml]: to be consistent with `dockerfile_path` and `container_image_name`, the default should be `""` too.

**Comment by geekodour** [comment]
@simonpasquier added the suggested changes :)

**Comment by simonpasquier** [comment]
:+1:

**Comment by simonpasquier** [comment]
@geekodour I've pushed the 0.5.0 tag including this PR :)

**Pull request merged**